src/data/subtype.lean

```diff
@@ -43,6 +43,10 @@ protected theorem forall' {q : ∀x, p x → Prop} :
 lemma ext_iff {a1 a2 : {x // p x}} : a1 = a2 ↔ (a1 : α) = (a2 : α) :=
 ⟨congr_arg _, subtype.ext⟩
 
+lemma heq_iff_coe_eq (h : ∀ x, p x ↔ q x) {a1 : {x // p x}} {a2 : {x // q x}} :
```

Comment:
```suggestion
@[simp] lemma heq_iff_coe_eq (h : ∀ x, p x ↔ q x) {a1 : {x // p x}} {a2 : {x // q x}} :
```

Reply:
If I do that then this lemma takes priority over the stronger `heq_iff_eq`. Note also that this is a conditional lemma, so probably shouldn't be simp